build.gradle

```diff
@@ -92,8 +92,22 @@ subprojects {
     tasks.check.dependsOn(tasks.jacocoTestReport)
     tasks.check.dependsOn(javadoc)
 
-    if (System.env.CIRCLE_TEST_REPORTS) {
-        test.reports.junitXml.destination = new File(System.env.CIRCLE_TEST_REPORTS, it.getName())
+    if (System.env.CIRCLECI) {
+        test.reports.junitXml.destination = new File(System.env.TESTS_DIR, it.getName())
+
+        task copyCiArtifactsToCircle {
```

Comment:
Chatted with Mark in person -- correct fix if we need artifacts would be to update to have the tasks write directly to target directory instead. However, in this case we don't actually need the artifacts, so we're just going to drop it. Updating PR now.